tests/integration_tests/test_export.py

```diff
@@ -0,0 +1,82 @@
+import os.path
+import logging
+import pytest
+
+from ludwig.train import train_cli
+from ludwig.export import export_savedmodel, export_neuropod
+from tests.integration_tests.utils import binary_feature
+from tests.integration_tests.utils import category_feature
+from tests.integration_tests.utils import generate_data
+from tests.integration_tests.utils import numerical_feature
+
+
+@pytest.mark.parametrize('export_type',
+                         ['savedmodel', 'neuropod'])
+def test_export(export_type, csv_filename, tmpdir):
```

Comment:
Can you refactor to have two separate tests? And place them in text_cli and use the _cli methods from export.py? Becuase we already know that the functions on their own work, testing the cli commands is want we wnat to make sure works fine (including the parsed args).

Reply:
Do you still want the `test_export.py`?  Or should I remove it?

Reply:
We probably don't need it I believe.